main.go

```diff
@@ -118,12 +121,20 @@ func main() {
 	jt = newJobTracker(*resourceManagerURL, *historyServerURL, *proxyServerURL)
 	go jt.Loop()
 
-	if err := jt.testLogsDir(); err != nil {
+	var err error
+	if err = jt.testLogsDir(); err != nil {
 		log.Printf("WARNING: Could not read yarn logs directory. Error message: `%s`\n", err)
 		log.Println("\tYou can change the path with --yarn-logs-dir=HDFS_PATH.")
 		log.Println("\tTo talk to HDFS, Timberlake needs to be able to access the namenode (--namenode-address) and datanodes.")
 	}
 
+	if metrics, err = statsd.New(*statsdAddress); err != nil {
+		log.Printf("WARNING: Could not initiate statsd client. Error message: `%s`\n", err)
+		log.Println("\tYou can change the path with --statsd-address=STATSD_ADDR.")
+	}
+	metrics.Namespace = "timberlake."
```

Comment:
Idea: Make this configurable? It's helpful sometimes for folks to prefix their metrics with other things that mean something in their world. :)